tests/unit/jobs/test_views.py

```diff
@@ -669,6 +669,7 @@ def test_get_challenge_submissions(self):
                 "status": self.submission.status,
                 "input_file": "http://testserver%s"
                 % (self.submission.input_file.url),
+                "submission_input_file": None,
```

Comment:
Can you follow similar format for checking this? i.e `self.submission.submission_input_file` ? @savish28 
Same in other places.
cc: @Ram81

Reply:
@KhalidRmb, Ideally since it is a file it should be "http://testserver%s"% (self.submission.submission_input_file.url), but since during initial submission of the file this would always be NONE. So should I change it to self.submission.submission_input_file or keep it as None?

Reply:
> @KhalidRmb, Ideally since it is a file it should be "http://testserver%s"% (self.submission.submission_input_file.url), but since during initial submission of the file this would always be NONE. So should I change it to self.submission.submission_input_file or keep it as None? Same is the case with stderr/stoud file they are also checked as None.

Makes sense. None is fine for now